src/stores/ServicesStore.js

```diff
@@ -338,6 +339,10 @@ export default class ServicesStore extends Store {
     this.filterNeedle = null;
   }
 
+  @action _resetStatus() {
+    this.actionStatus = [];
```

Comment:
The Store class actually provides `resetStatus()` to take care of this. I know that I'm doing the same thing in UserStore which I need to refactor soon. 

https://github.com/meetfranz/franz/blob/bb2a79a5d10ef2caab482a7c77b0f82ecfcdd1d0/src/stores/lib/Store.js#L41